request/request.go

```diff
@@ -128,3 +128,13 @@ func GetExtraResources(req *v1beta1.RunFunctionRequest) (map[string][]resource.E
 	}
 	return out, nil
 }
+
+func GetCredentials(req *v1beta1.RunFunctionRequest) (map[string][]map[string][]byte, error) {
```

Comment:
What's the thinking behind this being a map of slice of map? I wouldn't have expected the slice.

Reply:
Indeed, my bad started from `GetExtraResources` procedure